docs/DeveloperGuide.md

```diff
@@ -659,6 +659,52 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <hr>
 
+**System**: FitBook
+
+**Use case**: UC05 - Adding/removing weight of a client
+
+**Actor**: User
+
+**MSS**
+
+1. User requests to see a list of all clients.
+1. FitBook displays a success message followed by the list of clients.
+1. User views the list and enters the `weight` command to add/delete a weight value of a client.
```

Comment:
![image](https://github.com/AY2324S2-CS2103T-T17-3/tp/assets/40649376/c8fff5dc-a349-4ebc-a796-7e8de9cd842b)

I think this might be too detailed for use cases